app/javascript/components/maintaining/SubmissionsSummaryTable.tsx

```diff
@@ -23,11 +36,15 @@ function SubmissionsSummaryTableRow({
   )
 }
 
-export function SubmissionsSummaryTable({ submissions }) {
+export function SubmissionsSummaryTable({
+  submissions,
+}: {
+  submissions: Submission[]
```

Comment:
If this is intended to be readonly (and I think it is. you don't want this table component to modify the prop's array which would cause potentially other higher up components that use the same value to break), mark it as such:
```suggestion
  submissions: readonly Submission[]
```